scripts/st2bootstrap-deb.sh

```diff
@@ -112,6 +112,34 @@ setup_args() {
         exit 1
     fi
   fi
+
+  # Python 3.6 package is not available in Ubuntu Xenial
+  # Installer can add it via 3rd party PPA based on user agreement
+  if [[ "$SUBTYPE" = 'xenial' ]]; then
+    sudo apt-get update > /dev/null 2>/dev/null
+    # check if python3.6 is available
+    if (! apt-cache show python3.6 2> /dev/null | grep 'Package:' > /dev/null); then
+      echo ""
+      echo "WARNING!"
+      echo "python3.6 package is required as a dependency for StackStorm and none of your software repositories provide it."
+      echo "We recommend switching to Ubuntu 18.04 LTS (Bionic) as a base OS."
+      echo ""
+      echo "Alternatively we'll try to add 3rd party python3.6 APT repository: https://launchpad.net/~deadsnakes/+archive/ubuntu/ppa."
+      echo "By continue you're aware of security risk associated with using unofficial 3rd party PPA repository."
+      echo ""
+      read -p "Press [y] to continue or [n] to cancel adding it: " choice
```

Comment:
I decided to not provide a dedicated installation flag for Ubuntu Xenial installer as everything could be already done with `yes |`  or `echo 'y' |` from the user side anyways.

Reply:
@amanda11 ^^ I believe that should answer your question about running installer in non-interactive mode.

Even if we add `--install-py3-from-insecure-ppa` flag for extra security, - it makes little sense as users would be able to use `yes | curl|bash` or `echo y | curl|bash` to workaround it.

Reply:
OK - so we just need to update the E2E tests to install with an echo "y" | installer then?
Fine with me.

Reply:
Yeah that's fine with me as well.